File: URI/C/multiplosde13.c
#include <stdio.h>

int main() {
    int esq, dir, aux, soma = 0, num;
    scanf("%d %d", &esq, &dir);
    if (esq>dir)
    {
        aux = dir;
        dir = esq;
        esq = aux;
    }
    num=esq;
    while (num<=dir)
    {
        if (num%13 != 0)
        {
            soma+=num;
        }
        num++;
    }
    printf("%d\n", soma);

 
    return 0;
}

File: URI/Lua/MultiplosDe13.lua
a = io.read("*number")
b = io.read("*number")
soma = 0
if b > a then
    aux = a
    a = b
    b = aux
end
for i=b, a do
    if i%13 ~= 0 then
        soma = soma + i
    end
end

print(soma)